packages/react/src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -35,7 +33,9 @@ const getIconSize = (size: SizeValue, sizeModifier: IconSizeModifier, v: IconVar
     },
   }
 
-  return modifiedSizes[size] && pxToRem(modifiedSizes[size][sizeModifier])
+  return sizeModifier && modifiedSizes[size] && modifiedSizes[size][sizeModifier]
```

Comment:
Actually, in my opinion the prev conditional is more readable. Is there a solid reason why it was changed?

Reply:
this change was necessary to remove branching at method's start:

```jsx
 if (!sizeModifier) {
    return v[`${size}Size`]
 }
```

Essentially, in this method we have two branches to handle, thus it is an expected thing that we need just one `if` for that. Previously there were two `if`s (at start of the method and at `return`) for, essentially, differentiate between the same two branches (`sizeModifier` is provided or not), and that has complicated the overall's function logic - and now there is only one.